test/testcafe/framework/shared/index.js

```diff
@@ -38,3 +39,14 @@ export const Constants = {
 export const getStateHandleFromSessionStorage = ClientFunction(() => {
   return window.sessionStorage.getItem('osw-oie-state-handle');
 });
+
+export const a11yCheck = async (t) => {
```

Comment:
This is the meat of the accessibility testing logic.

Across our other products, we adhere to Section 508, WCAG21A, and WCAG21AA standards.